.cspell/en-words.txt

```diff
@@ -24,6 +24,7 @@ backoff
 caml
 cartservice
 cassandra
+CICS
```

Comment:
Hi @rrschulze! Thanks for your efforts here, but we actually reserve the `.cspell` file for words that are common across our docs. To appease the spelling linter for your post, you can add a `cSpell:ignore` field to the Hugo front matter. I'll show you where in a separate comment.